lib/src/mobile_ui/61/page_61.dart

```diff
@@ -0,0 +1,204 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+class Page61 extends StatelessWidget {
+  const Page61({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        backgroundColor: Colors.white,
+        foregroundColor: Colors.black,
+        centerTitle: false,
+        elevation: 0,
+        title: Container(
+          width: 122,
+          height: 15,
+          decoration: ShapeDecoration(
+            color: Color(0xFFA8A8A8),
+            shape: RoundedRectangleBorder(
+              borderRadius: BorderRadius.circular(50),
+            ),
+          ),
+        ),
+        actions: [
+          Container(
+            width: 30,
+            height: 30,
+            margin: EdgeInsets.only(
+              right: 24,
+            ),
+            decoration: BoxDecoration(
+              color: Color(0xFFC4C4C4),
+              shape: BoxShape.circle,
+            ),
+          ),
+        ],
+      ),
+      body: SingleChildScrollView(
+        child: Column(
+          children: [
+            SingleChildScrollView(
```

Comment:
You can use listview builder here and wrap with sizedbox.

Reply:
Okay